extension/intents/scroll/scrollHelper.js

```diff
@@ -2,8 +2,13 @@
 
 const scrollAmount = 0.9;
 
+const url = location.href;
+const pdf = url.slice(-4) === ".pdf";
```

Comment:
This works for the one example, but generally it's not good to try to inspect a URL to understand the type of the document. @danielamormocea used `document.contentType`, which is a good way to check the content type... but that's actually not the right way to fix this anyway, as other sites like https://bugzilla.mozilla.org/show_bug.cgi?id=343205 or a Google Doc also aren't scrollable.